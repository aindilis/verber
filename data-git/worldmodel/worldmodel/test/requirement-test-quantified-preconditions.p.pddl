(define
 (problem REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS)
 (:domain REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS)
 (:objects obj2 obj1 - type1)
 (:init
  (pred1 obj1))
 (:goal
  (and
   (pred1 obj2))))