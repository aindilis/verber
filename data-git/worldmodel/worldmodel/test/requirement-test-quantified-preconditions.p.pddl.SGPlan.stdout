#
# Copyright (C) 2006, Board of Trustees of the University of Illinois.
#
# The program is copyrighted by the University of Illinois, and should
# not be distributed without prior approval.  Commercialization of this
# product requires prior licensing from the University of Illinois.
# Commercialization includes the integration of this code in part or
# whole into a product for resale.
#
#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
# Author: C. W. Hsu, B. W. Wah, R. Y. Huang, Y. X. Chen
#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-


Parsing domain file
 domain 'REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS' defined
Parsing problem file
 problem 'REQUIREMENT-TEST-QUANTIFIED-PRECONDITIONS' defined



checking for cyclic := effects --- OK.



